source/exceptions.tex

```diff
@@ -1128,8 +1128,11 @@
 whose continuation is not a handle to a coroutine
 whose promise type has an \tcode{unhandled_stopped} member function.
 
+\item%
+when \tcode{std::execution::get_parallel_scheduler} is called and
+\tcode{std::execution::system_context_replace\-ability::query_parallel_scheduler_backend()}
+returns a null pointer value\iref{exec.par.scheduler}.
```

Comment:
The paper says
> returns `nullptr`

Seems like a fixup commit is needed, assuming this is purely editorial.

Reply:
This change from the paper's wording is intentional, you can't return the keyword "nullptr" (as seen by the caller).